### DISCUSSION TITLE
Certificado

### DISCUSSION THREAD

**Discussion by Sebasfullstack** [Launch-X-Latam/OnBoarding#15] [Q&A]
Cordial saludo, al finalizar el curso obtendremos certificado?

**Comment by devonent** [comment]
No, pero creo que si nos darán un reconocimiento (o algo parecido) de nuestra participación.

**Comment by JosafatJimenezB** [comment]
@Sebasfullstack no pero puedes acceder al **cloud skills challenge** con el cual obtendrás la certificación az-900 que es sobre el manejo de la nube de microsoft **azure** y te la recomiendo mucho!!

**Comment by gumodi35** [comment]
Mencionaron un certificado de participación, por completar todo el programa, eso.
